.changelog/14649.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+scheduler: Fixed a bug where MRD jobs wouldn't get deployments even if they previously had one
```

Comment:
I don't think "MRD" is in any of the user-facing documentation, that's just our internal acronym for it:

```suggestion
scheduler: Fixed a bug where multi-region jobs wouldn't get deployments even if they previously had one
```